client/app/scripts/superdesk/ui/ui.js

```diff
@@ -1142,8 +1142,83 @@ define([
         };
     }
 
-    return angular.module('superdesk.ui', ['superdesk.dashboard.world-clock'])
+    /*
+     * Splitter widget, allows user to dinamicaly
+     * resize monitoring and authoring screen
+     *
+     */
+    splitterWidget.$inject = ['superdesk', '$timeout'];
+    function splitterWidget(superdesk, $timeout) {
+        return {
+            link: function(scope, element) {
+                var workspace = element,
+                    authoring = element.next('#authoring-container');
+
+                /*
+                 * If custom sizes are defined, preload them
+                 */
+                if (superdesk.monitoringWidth && superdesk.authoringWidth) {
+                    workspace.width(superdesk.monitoringWidth);
+                    authoring.width(superdesk.authoringWidth);
+                }
 
+                /*
+                 * If authoring is not initialized,
+                 * wait, and initialize it again
+                 *
+                 * This issue is observed when you are
+                 * switching from settings back to monitoring
+                 */
+                if (!authoring.length) {
+                    $timeout(function () {

```

Comment:
you can set third param to false not to trigger another digest if you only change css